src/components/login-view/login-view.tsx

```diff
@@ -34,37 +35,38 @@ const LoginView = ({ onLoggedIn }: LoginProps) => {
       .catch((err: Error) => console.error(err));
   };
   return (
-    <div>
-      <h2>Login</h2>
-      <form onSubmit={handleSubmit}>
-        <div>
-          <label>
-            Username:
-            <input
-              type="text"
-              value={username}
-              onChange={(e) => setUsername(e.target.value)}
-              required
-              minLength={3}
-            />
-          </label>
+    <Col sm={6}>
+      <Form
```

Comment:
Make use of reusable components, it's not always good to have such huge components. better to put forms and and inputs fields into a reusable common component then reference them